SwiftCharts/Views/ChartAreasView.swift

```diff
@@ -51,6 +51,10 @@ open class ChartAreasView: UIView {
     }
     
     fileprivate func show(path: UIBezierPath) {
+        guard colors.count > 0 else {
```

Comment:
It's only to remove completely the subscripting. With the guard as it was before, it was safe, but you still could reorganize the code, accidentally disable the guard (etc) and then you have the unsafe subscripting back. This way it's not possible to mess it up.